ui/src/components/DashboardInventory.js

```diff
@@ -0,0 +1,150 @@
+//@flow
+
+import React from 'react';
+import { useQuery } from 'react-query';
+import styled, { css } from 'styled-components';
+import { Card, Loader } from '@scality/core-ui';
+import {
+  spacing,
+  fontSize,
+  fontWeight,
+} from '@scality/core-ui/dist/style/theme';
+
+import {
+  useAlertLibrary,
+  useHighestSeverityAlerts,
+  highestAlertToStatus,
+} from '../containers/AlertProvider';
+import { intl } from '../translations/IntlGlobalProvider';
+import { usePlatformLibrary } from '../containers/PlatformLibraryProvider';
+import { useAuth } from '../containers/AuthProvider';
+import { useConfig } from '../containers/ConfigProvider';
+import { PageSubtitle } from '../components/style/CommonLayoutStyle';
+import { STATUS_WARNING, STATUS_CRITICAL } from '../constants';
+
+const InventoryContainer = styled.div`
+  padding: 0px ${spacing.sp2};
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
+`;
+
+const CardsWrapper = styled.div`
+  display: flex;
+  justify-content: space-around;
+  font-size: ${fontSize.base};
+  margin: ${spacing.sp4} 0px;
+`;
+
+const InventoryIcon = styled.i`
+  font-size: ${fontSize.larger};
+  margin-right: ${spacing.sp4};
+  ${(props) => {
+    switch (props.status) {
+      case STATUS_WARNING:
+        return css`
+          color: ${props.theme.brand.statusWarning};
```

Comment:
FYI: we've removed the brand layer from the theme. So from now on, we can directly retrieve the color from the theme object.
```suggestion
          color: ${props.theme.statusWarning};
```